synapse/lib/certdir.py

```diff
@@ -1053,6 +1053,9 @@ def _loadKeyPath(self, path):
     def _loadP12Path(self, path):
         byts = self._getPathBytes(path)
         if byts:
+            # This API is deprecrated by PyOpenSSL and will need to be rewritten if pyopenssl is
+            # updated from v21.x.x. The APIs that use this are not directly exposed via the
+            # easycert tool currently, and are only used in unit tests.
```

Comment:
The corresponding cryptography library API returns a different object type than the pyopenssl library does; so changing that out isn't 100% free from an API perspective.